verify/verify/src/lib.rs

```diff
@@ -1,11 +1,26 @@
 #![deny(warnings)]
-#![feature(rust_2018_preview, avx512_target_feature, abi_vectorcall)]
+#![cfg_attr(
```

Comment:
Since this crate is only used for testing, I think we could have a global `#![cfg(test)]` to just disable everything in the crate when not compiling tests. That should reduce the `cfg(test)` duplication.